deploy/examples/multus-validation.yaml

```diff
@@ -1,11 +1,50 @@
-#################################################################################################################
+####################################################################################################
+# This manifest contains a Kubernetes Job and supporting definitions for running Rook's Multus
+# validation tool. The Multus validation tool's help text is copied below. Modify the Job
+# definition in this manifest based on the needs of your cluster.
+####################################################################################################
+# THE BELOW HELP TEXT IS AUTO-GENERATED BY `make docs`
+#
+#Run a validation test that determines whether the current Multus and system
+#configurations will support Rook with Multus.
+#
+#This should be run BEFORE Rook is installed.
```

Comment:
Can this be amended to have the commented lines be in the `# ` format (rather than just `#`) followed by other example manifests?

```suggestion
# Run a validation test that determines whether the current Multus and system
# configurations will support Rook with Multus.
#
# This should be run BEFORE Rook is installed.
```